src/main/java/uk/gov/hmcts/divorce/document/print/LetterPrinter.java

```diff
@@ -38,7 +38,7 @@ public void sendLetters(final CaseData caseData,
 
         var currentPacks = Optional.ofNullable(caseData.getDocuments().getLetterPacks()).orElse(new ArrayList<>());
         var documents = letters.stream().map(this::documentFromLetter).map(this::toListValue).toList();
-        var letterPack = LetterPack.builder().letters(documents).build();
+        var letterPack = LetterPack.builder().letters(documents).recipientAddress(applicant.getAddress()).build();
```

Comment:
`applicant.getCorrespondenceAddressWithoutConfidentialCheck()` It will need to be a string